pkg/plugins/ifaces.go

```diff
@@ -99,6 +99,8 @@ type Licensing interface {
 	Edition() string
 
 	Path() string
+
+	AppUrl() string
```

Comment:
Nit: I think this should be `AppURL` instead?

Reply:
yeah, I think so :)  fixed in 3ec2129ccc2acc61d7588919fe76a0a5b5f45253